internal/checker/cloudformation/rules/encryption/verify_at_rest_dax_is_encrypted.rego

```diff
@@ -0,0 +1,49 @@
+# METADATA
+# title: "Verify at rest DAX is encrypted"
+# description: "Automate verification to ensure AWS DAX clusters maintain encryption for data at rest."
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dax-cluster.html
+# custom:
+#   ID: CB_CFT_38
+#   Severity: High
+package lib.cloudformation.CB_CFT_38
+
+import future.keywords.in
+
+resource := "AWS::DAX::Cluster"
+
+is_valid {
+	some resources in input.Resources
+	resources.Type == resource
+}
+
```

Comment:
just check
pass[properties] {
    is_valid
    some resources in input.Resources
    properties := resources.Properties
    properties.SSESpecification.SSEEnabled == true

}